CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,43 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers in the Kubecost community, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+We are committed to making participation in the Kubecost community a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality.
```

Comment:
I would recommend against the list here and go with just, "We are committed to making participation in the Kubecost community a harassment-free experience for everyone."

By enumerating the list, we're by definition leaving out other things (e.g. skin color, viewpoint, political affiliation, all manner of other kinds of beliefs, etc.). So if it were up to me, I'd just not have a list in the first place. By definition "everyone" covers it.